arches_for_science/test_settings.py

```diff
@@ -1,39 +0,0 @@
-import os
-
-# Depends on /arches being on the python path, as arches.tests is not importable
-from tests.test_settings import *
-
-APP_NAME = "arches_for_science"
-APP_ROOT = os.path.dirname(__file__)
-INSTALLED_APPS = (
-    "webpack_loader",
-    "django.contrib.admin",
-    "django.contrib.auth",
-    "django.contrib.contenttypes",
-    "django.contrib.sessions",
-    "django.contrib.messages",
-    "django.contrib.staticfiles",
-    "django.contrib.gis",
-    "arches",
-    "arches.app.models",
-    "arches.management",
-    "arches_templating",
-    "guardian",
-    "captcha",
-    "revproxy",
-    "corsheaders",
-    "oauth2_provider",
-    "django_celery_results",
-    "arches_for_science",
-    "pgtrigger",
-    # "debug_toolbar"
-)
-ROOT_URLCONF = "arches_for_science.urls"
-
-LOCALE_PATHS = [os.path.join(APP_ROOT, "locale")]
-
-FUNCTION_LOCATIONS = ["arches_for_science.pkg.extensions.functions"]
```

Comment:
Noticed that `LOCALE_PATHS` and `FUNCTION_LOCATIONS`[1] are different here than in the new test settings file. Consider having the new test settings file do this first:
```
from arches_for_science.settings import *
```

Do you want that change in the project template?
***

[1] `FUNCTION_LOCATIONS` might be moot after archesproject/arches#10613 and archesproject/arches#11016, so not a big deal, mostly just interested in the right principle of settings inheritance.

Reply:
I've removed this file